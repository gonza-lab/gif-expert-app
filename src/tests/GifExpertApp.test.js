import GifExpertApp from '../GifExpertApp';
import React from 'react';
import { shallow } from 'enzyme';

describe('Pruebas en GifExpertApp', () => {
  let wrapper = shallow(<GifExpertApp />);

  test('should ', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
