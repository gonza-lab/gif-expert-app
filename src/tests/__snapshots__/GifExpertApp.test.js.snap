// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en GifExpertApp should  1`] = `
<div>
  <h2>
    GifExpertApp
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category="One Punch"
      key="One Punch"
    />
  </ol>
</div>
`;
